config.toml

```diff
@@ -34,12 +34,13 @@ googleFonts = "https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,1
 [permalinks]
 blog = "/blog/:slug/"
 
+## CSS Plugins
+[[params.plugins.css]]
+# Font Awesome
+link = "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.7.2/css/all.min.css"
```

Comment:
The CDN was previously switched out for the kit because the kit automatically subsets (only loads the icons used on the site). The kit has a page views limit for the free account (10k/month) and we've reached over 6k views (yay!) so far this month. This change was made to ensure that the icons will always be visible.